### PR TITLE
add stats and change heap

### PR DESCRIPTION
**Pull request by itielshwartz**

### CODE REVIEW

**Comment by hdm23061993** [comment]
How to view these stats?

File: pkg/dashboard/static/list-view.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ function loadChartsList() {
     const chartsCards = $("#installedList .body")
     chartsCards.empty().append("<div><span class=\"spinner-border spinner-border-sm\" role=\"status\" aria-hidden=\"true\"></span> Loading...</div>")
     $.getJSON("/api/helm/charts").fail(function (xhr) {
+        sendStats('Get charts', {'status': 'failed'});
```
Comment (undera) [inline pkg/dashboard/static/list-view.js:7]: It's rather "get releases" here than "charts".  The API call is named by mistake, I plan to address this in #25 
Minor thing, up to you.

File: pkg/dashboard/static/scripts.js [hunk 2]
```diff
@@ -115,6 +116,11 @@ myAlert.addEventListener('close.bs.alert', event => {
     $("#errorAlert").hide()
 })
 
+function sendStats(name, prop){
```
Comment (undera) [inline pkg/dashboard/static/scripts.js:119]: This function should reside in `analytics.js`

File: main.go [hunk 3]
```diff
@@ -11,20 +11,16 @@ import (
...
 
-var (
-	version = "dev"
-	commit  = "none"
-	date    = "unknown"
-)
-
 type options struct {
-	Version    bool   `long:"version" description:"Show tool version"`
+	Version    string `long:"version" description:"Show tool version"  default:"0.0.0"`
```
Comment (undera) [inline main.go:15]: The `options` struct is the list of CLI flags. IMO it's wrong to mix the responsibility of this struct with status display.
One-time need of development not worth confusing users when they call `--help`. If you look for dev needs overrides, use the `HD_`-prefixed env variables in appropriate places.

File: pkg/dashboard/server.go [hunk 4]
```diff
@@ -35,10 +35,11 @@ func (s Server) StartServer() (string, utils.ControlChan) {
 		log.Errorf("Failed to check that Helm is operational, cannot continue. The error was: %s", err)
 		os.Exit(1) // TODO: propagate error instead?
 	}
-
+	isDevModeWithAnalytics := s.Version == "0.0.0" && os.Getenv("enableDevAnalytics") == "true"
```
Comment (undera) [inline pkg/dashboard/server.go:38]: Env variable should be all-caps and have `HD_` prefix, it's a sort of project standard.
The `s.Version == "0.0.0"` condition is not needed, the second part of condition handles all cases perfectly.

**Review by undera — CHANGES_REQUESTED** [review]
Please address my comments

**Comment by undera** [comment]
> How to view these stats?

Sorry, it is not possible to view these stats publicly.

File: pkg/dashboard/server.go [hunk 5]
```diff
@@ -136,13 +137,13 @@ func checkUpgrade(d *subproc.StatusInfo) {
 
 	v1, err := version.NewVersion(d.CurVer)
 	if err != nil {
-		log.Warnf("Failed to parse version: %s", err)
+		log.Warnf("Failed to parse CurVer: %s", err)
```
[the remark this answers is not in the document]
Comment (itielshwartz) [reply pkg/dashboard/server.go:140]: Sorry i think i don't understand you, I think I fixed the confusion that was there before me

File: main.go (hunk 3 above)
Comment (itielshwartz) [reply main.go:15]: so why do we need the version as part of the cli?

File: main.go [hunk 6]
```diff
@@ -11,20 +11,16 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var (
```
[the remark this answers is not in the document]
Comment (itielshwartz) [reply main.go:14]: not sure i undersand

File: pkg/dashboard/static/list-view.js (hunk 1 above)
Comment (itielshwartz) [reply pkg/dashboard/static/list-view.js:7]: what do you suggest?

File: main.go (hunk 6 above)
[the remark this answers is not in the document]
Comment (undera) [reply main.go:14]: We use https://goreleaser.com/customization/templates/ to set the version information for the project. It requires these variables to exist and receive the version replacement.

File: pkg/dashboard/server.go (hunk 5 above)
[the remark this answers is not in the document]
Comment (undera) [reply pkg/dashboard/server.go:140]: What's the confusion? The warn messages were explaining which exact field of struct has failed to parse. So when you would see it in log, you will know from which code block it originates. 
You made it less specific, duplicate, thus more confusing.

[the remark this answers is not in the document]
Comment (undera) [reply pkg/dashboard/server.go:140]: OMG, my bad, you did exactly opposite, you improved it. My apologies.

File: main.go (hunk 3 above)
Comment (itielshwartz) [reply main.go:15]: oh i understand! my bad here

**Pull request merged**